src/core/gosling-track-model.ts

```diff
@@ -312,15 +313,21 @@ export class GoslingTrackModel {
             return true;
         }
 
-        const { operation, condition } = spec.visibility;
-        if (condition.zoomLevel && currentStage.zoomLevel) {
-            return logicalComparison(currentStage.zoomLevel, operation, condition.zoomLevel) === 1;
-        } else if (condition.height && spec.height) {
-            return logicalComparison(spec.height, operation, condition.height) === 1;
-        } else if (typeof condition.width === 'number' && spec.width) {
-            return logicalComparison(spec.width, operation, condition.width) === 1;
+        const { operation, measure, threshold } = spec.visibility;
+
+        let compareValue: number | undefined = undefined;
+
+        if (measure == 'zoomLevel') {
```

Comment:
Minor comment: Lets use triple equals (`===` or `!==`) everywhere for safer type checks.